src/test/java/com/juniorstart/juniorstart/service/UserServiceTest.java

```diff
@@ -30,59 +31,61 @@
 
 /** Represents an user service.
  * @author Dawid Wit
- * @version 1.0
- * @since 1.0
+ * @version 1.2
+ * @since 1.1
  */
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @RunWith(MockitoJUnitRunner.class)
 public class UserServiceTest {
-
-
-    UserDao userDao = Mockito.mock(UserDao.class);
-    UserService userService = new UserService(userDao);
+    @Mock
+    UserDao userDao;
+    @InjectMocks
+    UserService userService;
     UUID uuid = UUID.randomUUID();
-
     User user;
     UserPrincipal userPrincipal;
-    Instant createdAt;
-
 
     /** Create user for further tests.
      */
-
     @Before
-    public void initialize() throws Exception {
-        createdAt = new SimpleDateFormat("yyyy-MM-dd").parse("2020-12-31").toInstant();
+    public void initialize() {
+        user = User.builder()
+                .privateId(uuid)
+                .publicId(10L)
+                .name("Test")
+                .age(18)
+                .hiddenFromSearch(false)
+                .email("test@test.com")
+                .imageUrl("test Url")
+                .emailVerified(true)
+                .password("Password")
+                .provider(AuthProvider.local)
+                .providerId("id").build();
 
-         user = User.builder().name("MockName").password("Password").publicId(12L).provider(AuthProvider.local).email("grzesiek12@gmail.com").emailVerified(true).privateId(uuid).build();
         userPrincipal = UserPrincipal.create(user);
-
-
+        Mockito.when(userDao.findByPrivateIdAndPassword(user.getPrivateId(), user.getPassword())).thenReturn(Optional.of(this.user));
     }
 
     @Test
-    public void should_getCurrentUser() throws Exception {
+    public void should_getCurrentUser()  {
 
         Mockito.when(userDao.findByPrivateId(ArgumentMatchers.any())).thenReturn(Optional.of(user));
         Assert.assertEquals(userService.getCurrentUser(userPrincipal).getEmail(),user.getEmail());
     }
 
-
     /** Test of correct data.
      */
     @Test
     public void testChangeEmail() {
         //Given
-        Optional<User> optional = Optional.of(this.user);
         User mockUser = this.user;
```

Comment:
1. Nie lepiej zamiast this.user użyć poprostu user?
2. Nie lepiej zainicjować klasy ChangeEmail w metodzie setUp ?